examples/sandbox/index.ts

```diff
@@ -16,7 +16,9 @@ import * as tallyHo from "@shapeshiftoss/hdwallet-tallyho";
 import * as trezorConnect from "@shapeshiftoss/hdwallet-trezor-connect";
 import { WalletConnectProviderConfig } from "@shapeshiftoss/hdwallet-walletconnect";
 import * as walletConnect from "@shapeshiftoss/hdwallet-walletconnect";
+import * as walletConnectv2 from "@shapeshiftoss/hdwallet-walletconnectv2";
```

Comment:
CI failing because of this line - `@shapeshiftoss/hdwallet-walletconnectv2` is not yet published.